pkg/kn/traffic/compute.go

```diff
@@ -242,12 +242,12 @@ func errorSumGreaterThan100(sum int) error {
 // - if provided traffic portion are integers
 // - if traffic as per flags sums to 100
 // - if traffic as per flags < 100, can remaining traffic be automatically directed
-func verifyInput(trafficFlags *flags.Traffic, svc *servingv1.Service, revisions []servingv1.Revision, mutation bool) error {
+func verifyInput(trafficFlags *flags.Traffic, svc *servingv1.Service, targets []servingv1.TrafficTarget, existingRevisions []servingv1.Revision, mutation bool) error {
 	// check if traffic is being sent to @latest tag
 	var latestRefTraffic bool
 
-	// number of existing revisions
-	var existingRevisionCount = len(revisions)
+	// number of existing targets
+	var existingRevisionCount = len(targets)
```

Comment:
would it make sense to rename the variable to `targetsCount` ?

Reply:
Yes, I'll change it